docs/adding_new_benchmark.md

```diff
@@ -0,0 +1,60 @@
+This is a detailed document of how to add a new benchmark in vSwarm.
+# Add Standalone Benchmarks in vSwarm
+## Python
+### 1. Add protobuf
+The first thing we need to do is to add protobuf in [vSwarm-proto](https://github.com/vhive-serverless/vSwarm-proto).
+
+First, clone [vSwarm-proto](https://github.com/vhive-serverless/vSwarm-proto), operations below should be taken in your local `vSwarm-proto/` folder.
+
+Add a new folder to `proto/` named by your benchmark `{bench}`, add file `{bench}.proto` to this `proto/{bench}/` folder, which should define the protobuf of your benchmark.
+
+In `{bench}.proto` file, always define your service as `Greeter` since this is the grpc service that will be invoked by the relay.
```

Comment:
It does not have to be Greeter. I.e. most of the standalone benchmarks have custom protocols. (https://github.com/vhive-serverless/vSwarm-proto/blob/main/proto/aes/aes.proto). This is actualy the reason why we have the relay. Otherwise, the invoker can be used to invoke the function without the relay.

```suggestion
In `{bench}.proto` file, define the needed grpc service. Note that invoker and most of vHive work with the simple [HelloWorld](https://grpc.io/docs/what-is-grpc/introduction/) service. However, you can implement arbitrary services for your function as the relay, sitting in between the invoker and your function, can be used to make translations. Refer to [aes.proto](https://github.com/vhive-serverless/vSwarm-proto/blob/main/proto/aes/aes.proto) and the corresponding [translation](https://github.com/vhive-serverless/vSwarm-proto/blob/main/grpcclient/aes_client.go) as an example of a custom service. 
```